src/block/Block.php

```diff
@@ -233,6 +233,14 @@ public function onNearbyBlockChange() : void{
 
 	}
 
+	/**
+	 * Called when an Entity falls on this block.
+	 */
+	public function onEntityFall(Entity $entity, float $fallDistance, Vector3 $fallPos): void
+	{
```

Comment:
This should be formatted according to the PM code style.

Reply:
It would also be useful to pass the entity's velocity in here (for stuff like slime blocks).

Reply:
Passing `fallPos` here seems pointless, since it's always going to be the block's own position.